README.md

```diff
@@ -49,11 +49,18 @@ This builds the plugin in `/build/rootfs/`
 `export SNAP_PATH=$GOPATH/src/github.com/intelsdi-x/snap/build`
 * Load the plugin and create a task, see example in [Examples](https://github.com/intelsdi-x/snap-plugin-collector-interface/blob/master/README.md#examples).
 
+Configuration parameters:
+- `proc_path` path to 'dev/net' file (helpful for running plugin in Docker container)
+
 ## Documentation
 
 ### Collected Metrics
 List of collected metrics is described in [METRICS.md](https://github.com/intelsdi-x/snap-plugin-collector-interface/blob/master/METRICS.md).
 
+Plugin has ability to read metrics from `dev/net` file.
```

Comment:
I find this sentence confusing because the plugin will always read the data from the `dev/net` file.